Formula/octomap.rb

```diff
@@ -3,6 +3,7 @@ class Octomap < Formula
   homepage "https://octomap.github.io/"
   url "https://github.com/OctoMap/octomap/archive/v1.9.5.tar.gz"
   sha256 "adf87320c4c830c0fd85fe8d913d8aa174e2f72d0ea64c917599a50a561092b6"
+  license "GPL-2.0"
```

Comment:
According to the [readme](https://github.com/OctoMap/octomap):

> License:
> 
> - octomap: [New BSD License](https://github.com/OctoMap/octomap/blob/devel/octomap/LICENSE.txt)
> - octovis and related libraries: [GPL](https://github.com/OctoMap/octomap/blob/devel/octovis/LICENSE.txt)

Is this an instance where `["GPL-2.0", "BSD-3-Clause"]` should be used? If not, I would think `BSD-3-Clause` is more applicable because it's what the `octomap` source is under.

Reply:
I think we only build `octomap` here and not `octovis`.

Reply:
Good callout, yeah, BSD-3-Clause is a better fit.